src/app/MessageDef/ReadRequestMessage.cpp

```diff
@@ -180,7 +180,11 @@ CHIP_ERROR ReadRequest::Parser::GetAttributePathList(AttributePathList::Parser *
     return err;
 }
 
+<<<<<<< HEAD:src/app/MessageDef/ReadRequest.cpp
 CHIP_ERROR ReadRequest::Parser::GetEventPaths(EventPaths::Parser * const apEventPaths) const
+=======
+CHIP_ERROR ReadRequestMessage::Parser::GetEventPathList(EventPathList::Parser * const apEventPathList) const
+>>>>>>> 3d86bbb3d (Rename several IM message with Message postfix):src/app/MessageDef/ReadRequestMessage.cpp
```

Comment:
This is not going to go well.

Reply:
Merge conflict...Thanks for catching it up. Fixed.